service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/StoragetransferConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package bio.terra.workspace.service.resource.controlled.flight.clone.bucket;
+
+import com.google.api.client.googleapis.util.Utils;
+import com.google.api.services.storagetransfer.v1.Storagetransfer;
+import com.google.api.services.storagetransfer.v1.StoragetransferScopes;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.BeanCreationException;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+public class StoragetransferConfiguration {
+  private static final Logger logger = LoggerFactory.getLogger(StoragetransferConfiguration.class);
+
+  @Bean
+  @Profile("!unit-test")
+  public GoogleCredentials getGoogleCredentials() {
```

Comment:
Should this (and fakeGoogleCredntials) be in a separate file, something like `GoogleCredentialsConfiguration`? If I'm trying to track down where an autowired `GoogleCredentials` object came from, I wouldn't expect it to be in `StoragetransferConfiguration`

Reply:
In IJ ultimate, you can just click on the symbol next to the bean injection site...but good point.